src/serac/physics/contact/contact_data.hpp

```diff
@@ -161,9 +168,10 @@ class ContactData {
   /**
    * @brief Update the current coordinates based on the new displacement field
    *
+   * @param u_shape Shape displacement vector
    * @param u Current displacement dof values
    */
-  void setDisplacements(const mfem::Vector& u);
+  void setDisplacements(const mfem::Vector& u_shape, const mfem::Vector& u);
```

Comment:
Is `setDisplacements` the right name for this? Is it setting the current configuration?